scripts/artifacts-building/containers/build-process.sh

```diff
@@ -89,20 +93,60 @@ git config --global user.email "rcbops@rackspace.com"
 git config --global user.name "RCBOPS gating"
 
 # Patch the roles
+# TODO(odyssey4me):
+# Remove the patcher process once the following have merged
+# and are available to RPC-O:
+# https://review.openstack.org/474734
+# https://review.openstack.org/474730
 cd containers/patches/
 patch_all_roles
 
+# If we have no pre-built python artifacts available, the whole
+# container build process will fail as it is unable to find the
+# right artifacts to use. To ensure that we can still do a PR test
+# when there are no python artifacts, we need to override a few
+# things.
+if ! python_artifacts_available; then
+    # As there are no wheels available for this release, we will
+    # need to enable developer_mode for the role install.
+    echo "developer_mode: yes" >> ${OA_OVERRIDES}
+
+    # As there are is not pre-build constraints file available
+    # we will need to use those from upstream.
```

Comment:
Nit: Should be "As there are not pre-built constraints files available we will need to use those from upstream."